packages/suite/src/views/dashboard/components/T2B1PromoBanner.tsx

```diff
@@ -272,7 +255,7 @@ export const T2B1PromoBanner = () => {
                     <TrezorSafe3Logo image="TREZOR_SAFE_PROMO_LOGO" />
                     <NextGenerationText>
```

Comment:
What about not use `font-size` in `NextGenerationText` and put new prop to `Text`? 
It should have these styles:
<img width="328" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/8c00b4bf-1d1c-4807-953a-eb231fc0575a">

WDYT?

Reply:
I have added the typography to the `<Text>` however I would need a help with applying it.